apps/tutorial-servo-can-control/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include "TutorialServo.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+#include "mbed.h"
+
+int main()
+{
+    CANBus can(CAN1_RX, CAN1_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
```

Comment:
these declarations are well done (since they are right at the top of your main function), make sure to declare your servo object as one of the first things you do